__tests__/index.test.js

```diff
@@ -101,7 +70,24 @@ describe("nc news", () => {
               .expect(400)
               .then((response) => {
                 expect(response.body.msg).toBe('Invalid request');
-              });
-          });
+            });
+        });
+    })
+    describe("/api/articles", ()=> {
+        test("200: serves an array of all articles sorted by date in desc order", () => {
+            return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({body}) => {
+                  expect(body.articles.length >= 1).toBe(true)
+                  expect(body.articles).toBeSortedBy("created_at", { descending: true})
+                  body.articles.forEach((article) => {
+                    expect(article).toMatchObject({
+                        article_id: expect.any(Number), title: expect.any(String), topic: expect.any(String), author: expect.any(String),
+                        created_at: expect.any(String), votes: expect.any(Number), article_img_url: expect.any(String), comment_count: expect.any(String)//number?
+                    })
```

Comment:
Interestingly comment_count does by default come back as a string, but you can change this to be a number within your sql query if you want to do this. 

The formatting here makes this a little difficult to read, each property should ideally be on its own line. I would recommend installing prettier for this as it will make your life a lot easier.